kernel/main.cpp

```diff
@@ -77,13 +85,9 @@ void kernel_main(void *boot_info, uint32_t magic) {
     // Get the CPU vendor and model data to print
     const char *vendor = cpu_get_vendor();
     const char *model = cpu_get_model();
-    kprintf(DBG_INFO "%s\n", vendor);
-    kprintf(DBG_INFO "%s\n", model);
+    kprintf(DBG_INFO "%s %s\n", vendor, model);
```

Comment:
What happens now that we're using graphical mode?

Reply:
There won't be any graphical logging, but it's not like we got much information that way anyways. I figured it would be alright until I got fonts working. Once that is in place, a framebuffer will be required to boot and we'll drop VGA text mode.

Reply:
Okay cool. I just wanted to make sure that this wouldn't interfere with anything else